### DISCUSSION TITLE
The block feature has disappeared in the new version. Will it no longer be supported in the future?

### DISCUSSION THREAD

**Discussion by xxnuo** [wavetermdev/waveterm#913] [General]
I quite liked that feature.
Without this feature, Wave is no different from other terminals, and it also suffers from lower efficiency due to Electron.
The new IDE features are indeed quite innovative 👍
But there are already similar products like https://www.terminal.icu/

**Comment by texastoland** [comment]
I asked the same in https://github.com/wavetermdev/waveterm/discussions/779#discussioncomment-10798706.

> But there are already similar products like [terminal.icu](https://www.terminal.icu/)

This looks great! But I think there's no English translation available? Also doesn't look open source.

**Reply by xxnuo** [reply]
I submitted a request for the English interface translation. 
This software is a commercial product with free basic features and paid advanced features.

**Reply by texastoland** [reply]
Can you share a link to follow? I saw https://host.terminal.icu/topic/668/2-0 but still nothing about translation 🥲

**Reply by texastoland** [reply]
@xxnuo Do you have any knowledge about [English support](https://www.yuque.com/u64817/isrom7/vqwkwtqbhwgou9k4#record:ZLdiuDTh9EGnHGG784rg32yUg9Nhd2W5)?

**Comment by tigros** [comment]
i also thought this was a very neat thing, strange to see it go. love to see it make a comeback, maybe optional.
